src/drag-and-drop/components/app.tsx

```diff
@@ -66,10 +66,38 @@ export const baseAuthoringProps = {
               title: "Image URL",
               type: "string",
               format: "uri"
+            },
+            imageWidth: {
+              title: "Image width",
+              type: "number"
+            },
+            imageHeight: {
+              title:"Image height",
+              type: "number"
+            },
+            itemLabel: {
+              title: "Label",
+              type: "string"
+            },
+            index: {
+              title: "Index",
+              type: "number"
+            },
+            itemValue: {
+              title: "Value",
+              type: "number"
+            },
+            targetMatch: {
+              title: "Target match:",
+              type: "number"
             }
           }
         }
       },
+      enableCheckAnswer: {
+        title: "Allow users to check answers",
+        type: "boolean",
+      },
```

Comment:
Same here, seems to be unused.

Reply:
This was specified in the mockup here https://www.pivotaltracker.com/story/show/178218427. I'll take it out and put it in the PR for that story.